packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/dedicatedCloud-datacenter-dashboard.html

```diff
@@ -190,9 +190,36 @@
                 <oui-tile-definition
                     data-term="{{:: 'dedicatedCloud_nsx_edge_level' | translate }}"
                     data-description="{{:: $ctrl.datacenter.model.edgesLevel }}"
+                    data-ng-if="!$ctrl.pollNsxTaskId"
```

Comment:
Why this line ?

Relative to : MANAGER-15423

Reply:
I guess it's because you can't resize when an operation is already pending